README.md

```diff
@@ -1,2 +1,37 @@
-# odh-manifests
-A repository for Open Data Hub Kustomize manifests extending upstream Kubeflow manifests
```

Comment:
I am not sure about the word "extending" ? Because not everything here makes it upstream and some ODH components are independent . Maybe re-word it to "Open Data Hub components Kustomize manifest repo"

Reply:
This is what the README *had before*...I am removing this, so no need to fix it:)